### PR TITLE
add deprecated warning on some filters

### PR DESCRIPTION
**Pull request by itdependsnetworks**
##### SUMMARY

Add deprecated warning to some filters, that confuse the average user.

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

ip_addr

##### ANSIBLE VERSION

```
N/A
```

##### ADDITIONAL INFORMATION

```
N/A
```

### CODE REVIEW

**Comment by maxamillion** [comment]
cc @sivel

File: lib/ansible/plugins/filter/ipaddr.py
Comment (sivel) [inline lib/ansible/plugins/filter/ipaddr.py]: We should use `display.deprecated` here, and if we are doing so, we also need to indicate in which version they will be deprecated.  We have a +4 version requirement on deprecations.  So if this goes into 2.6, they could be removed in 2.10.

**Review by sivel — DISMISSED** [review]

File: lib/ansible/plugins/filter/ipaddr.py
Comment (itdependsnetworks) [reply lib/ansible/plugins/filter/ipaddr.py]: Thanks, just being a copy paste monkey without looking at all the methods. Will change and mention version as well.

**Comment by itdependsnetworks** [comment]
how do you remove needs review?

**Comment by bcoca** [comment]
@sivel needs to re review and approve

File: lib/ansible/plugins/filter/ipaddr.py
Comment (sivel) [inline lib/ansible/plugins/filter/ipaddr.py:663]: `display.deprecated` takes a `version` kwarg that defines the version that the functionality will be removed in.  Please use this as it will help us audit the code later to find items needing removed.

**Comment by sivel** [comment]
My apologies for taking so long.  I am traveling internationally right now and not working frequently.  I've added another comment.

I'm going to go ahead and "dismiss" my review for now, so that I don't necessarily hold things up in my absence.

**Comment by itdependsnetworks** [comment]
@sivel any chance to take a look? Came up again today that this is not what network engineers expect to happen

**Comment by itdependsnetworks** [comment]
Does not seem to be interest from core maintainers, closing

**Pull request closed without merging**
